vic/drivers/classic/src/read_vegparam.c

```diff
@@ -371,28 +372,28 @@ read_vegparam(FILE  *vegparam,
     }
 
     // Determine if we have bare soil
-    if (temp[0].Cv_sum > 1.0) {
-        log_warn("Cv exceeds 1.0 at grid cell %d, fractions being "
-                 "adjusted to equal 1", gridcel);
+    if (Cv_sum > 1.0) {
```

Comment:
Wouldn't this be a good opportunity to use assert_close_double()?  You might be able combine lines 375 - 391 into a single rescaling block.